exporter/uptraceexporter/exporter.go

```diff
@@ -119,6 +99,30 @@ func (e *traceExporter) pushTraceData(ctx context.Context, traces pdata.Traces)
 	return nil
 }
 
+func (e *traceExporter) Start(_ context.Context, _ component.Host) error {
```

Comment:
This can be hidden `start` correct?

Reply:
I do not see a reason it shouldn't be. I just kept the pattern that was implemented with their "Shutdown"